### ISSUE TITLE
[Bug] Alerting profiles event_type_filters change every time.

### ISSUE THREAD

**Issue by robertchgo** [dynatrace-oss/terraform-provider-dynatrace#113]
Similar to https://github.com/dynatrace-oss/terraform-provider-dynatrace/issues/61 but for event type filters.

```
~ resource "dynatrace_alerting_profile" "NAME" {
        id           = "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
        # (2 unchanged attributes hidden)

      ~ event_type_filters {

          ~ predefined_event_filter {
              ~ event_type = "PROCESS_HIGH_GC_ACTIVITY" -> "OSI_HIGH_CPU"
                # (1 unchanged attribute hidden)
            }
        }
      ~ event_type_filters {

          ~ predefined_event_filter {
              ~ event_type = "PROCESS_MEMORY_RESOURCE_EXHAUSTED" -> "OSI_LOW_DISK_SPACE"
                # (1 unchanged attribute hidden)
            }
        }
      ~ event_type_filters {

          ~ predefined_event_filter {
              ~ event_type = "OSI_HIGH_CPU" -> "PROCESS_MEMORY_RESOURCE_EXHAUSTED"
                # (1 unchanged attribute hidden)
            }
        }
      ~ event_type_filters {

          ~ predefined_event_filter {
              ~ event_type = "OSI_LOW_DISK_SPACE" -> "PROCESS_HIGH_GC_ACTIVITY"
                # (1 unchanged attribute hidden)
            }
        }

        # (3 unchanged blocks hidden)
    }
```
 is it possible to apply the same fix that was applied in Issue #61 to event_type_filters or a possible solution for this?

**Comment by Dynatrace-Reinhard-Pilz** [comment]
Fixed with todays upcoming release

**Comment by Dynatrace-Reinhard-Pilz** [comment]
This issue has been fixed with [Release v1.11.0](https://github.com/dynatrace-oss/terraform-provider-dynatrace/releases/tag/v1.11.0).

**Comment by Yuvaraj-mani-mns** [comment]
similar to this i'm getting like i can't able to create both custom filter & predefined, always predefined filter exists.
 
when i apply terraform apply again and again custom filter creating, see below output

> terraform apply
dynatrace_alerting.alert-profile["profile2"]: Refreshing state... 
dynatrace_alerting.alert-profile["profile1"]: Refreshing state... 

Terraform used the selected providers to generate the following execution plan. Resource actions are indicated with the following symbols: 
  ~ update in-place

Terraform will perform the following actions:

  # dynatrace_alerting.alert-profile["profile1"] will be updated in-place
  ~ resource "dynatrace_alerting" "alert-profile" {
        id              = "vu9U3hXa3q0AAAABABhidWlsdGluOmFsZXJ0aW5nLnByb2ZpbGUABnRlbmFudAAGdGVuYW50ACQ1NDI0YjU4ZS0xNWViLTMyMTQtYTUxZi03ZTA5MTRlYWYyYWW-71TeFdrerQ"
      + management_zone = "test-mz-1"
        name            = "alert-profile-1"
        # (1 unchanged attribute hidden)

      ~ filters {
          ~ filter {
              + custom {
                  + description {
                      + case_sensitive = false
                      + enabled        = true
                      + negate         = false
                      + operator       = "BEGINS_WITH"
                      + value          = "value123"
                    }
                  + metadata {
                      + items {
                          + filter {
                              + key    = "value-1"
                              + negate = false
                              + value  = "valu-2"
                            }
                          + filter {
                              + key    = "value-11"
                              + negate = false
                              + value  = "valu-22"
                            }
                        }
                    }
                  + title {
                      + case_sensitive = false
                      + enabled        = true
                      + negate         = false
                      + operator       = "CONTAINS"
                    }
                }

                # (1 unchanged block hidden)
            }
        }
    }

  # dynatrace_alerting.alert-profile["profile2"] will be updated in-place
  ~ resource "dynatrace_alerting" "alert-profile" {
        id              = "vu9U3hXa3q0AAAABABhidWlsdGluOmFsZXJ0aW5nLnByb2ZpbGUABnRlbmFudAAGdGVuYW50ACQ5ZjI1MjU1MS0zMDNmLTMzMDMtOWY5MC01ZmUyNjViOTcyNzS-71TeFdrerQ"
      + management_zone = "test-mz-2"
        name            = "alert-profile-2"
        # (1 unchanged attribute hidden)
    }

Plan: 0 to add, 2 to change, 0 to destroy.

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

dynatrace_alerting.alert-profile["profile2"]: Modifying... [id=vu9U3hXa3q0AAAABABhidWlsdGluOmFsZXJ0aW5nLnByb2ZpbGUABnRlbmFudAAGdGVuYW50ACQ5ZjI1MjU1MS0zMDNmLTMzMDMtOWY5MC01ZmUyNjViOTcyNzS-71TeFdrerQ]
dynatrace_alerting.alert-profile["profile1"]: Modifying... [id=vu9U3hXa3q0AAAABABhidWlsdGluOmFsZXJ0aW5nLnByb2ZpbGUABnRlbmFudAAGdGVuYW50ACQ1NDI0YjU4ZS0xNWViLTMyMTQtYTUxZi03ZTA5MTRlYWYyYWW-71TeFdrerQ]
dynatrace_alerting.alert-profile["profile2"]: Modifications complete after 0s [id=vu9U3hXa3q0AAAABABhidWlsdGluOmFsZXJ0aW5nLnByb2ZpbGUABnRlbmFudAAGdGVuYW50ACQ5ZjI1MjU1MS0zMDNmLTMzMDMtOWY5MC01ZmUyNjViOTcyNzS-71TeFdrerQ]
dynatrace_alerting.alert-profile["profile1"]: Modifications complete after 0s [id=vu9U3hXa3q0AAAABABhidWlsdGluOmFsZXJ0aW5nLnByb2ZpbGUABnRlbmFudAAGdGVuYW50ACQ1NDI0YjU4ZS0xNWViLTMyMTQtYTUxZi03ZTA5MTRlYWYyYWW-71TeFdrerQ]

Apply complete! Resources: 0 added, 2 changed, 0 destroyed.
P> terraform apply
dynatrace_alerting.alert-profile["profile2"]: Refreshing state... [id=vu9U3hXa3q0AAAABABhidWlsdGluOmFsZXJ0aW5nLnByb2ZpbGUABnRlbmFudAAGdGVuYW50ACQ5ZjI1MjU1MS0zMDNmLTMzMDMtOWY5MC01ZmUyNjViOTcyNzS-71TeFdrerQ]
dynatrace_alerting.alert-profile["profile1"]: Refreshing state... [id=vu9U3hXa3q0AAAABABhidWlsdGluOmFsZXJ0aW5nLnByb2ZpbGUABnRlbmFudAAGdGVuYW50ACQ1NDI0YjU4ZS0xNWViLTMyMTQtYTUxZi03ZTA5MTRlYWYyYWW-71TeFdrerQ]

Terraform used the selected providers to generate the following execution plan. Resource actions are indicated with the following symbols: 
  ~ update in-place

Terraform will perform the following actions:

  # dynatrace_alerting.alert-profile["profile1"] will be updated in-place
  ~ resource "dynatrace_alerting" "alert-profile" {
        id              = "vu9U3hXa3q0AAAABABhidWlsdGluOmFsZXJ0aW5nLnByb2ZpbGUABnRlbmFudAAGdGVuYW50ACQ1NDI0YjU4ZS0xNWViLTMyMTQtYTUxZi03ZTA5MTRlYWYyYWW-71TeFdrerQ"
      + management_zone = "test-mz-1"
        name            = "alert-profile-1"
        # (1 unchanged attribute hidden)

      ~ filters {
          ~ filter {
              + custom {
                  + description {
                      + case_sensitive = false
                      + enabled        = true
                      + negate         = false
                      + operator       = "BEGINS_WITH"
                      + value          = "value123"
                    }
                  + metadata {
                      + items {
                          + filter {
                              + key    = "value-1"
                              + negate = false
                              + value  = "valu-2"
                            }
                          + filter {
                              + key    = "value-11"
                              + negate = false
                              + value  = "valu-22"
                            }
                        }
                    }
                  + title {
                      + case_sensitive = false
                      + enabled        = true
                      + negate         = false
                      + operator       = "CONTAINS"
                      + value          = "value345"
                    }
                }

                # (1 unchanged block hidden)
            }
        }
    }

  # dynatrace_alerting.alert-profile["profile2"] will be updated in-place
  ~ resource "dynatrace_alerting" "alert-profile" {
        id              = "vu9U3hXa3q0AAAABABhidWlsdGluOmFsZXJ0aW5nLnByb2ZpbGUABnRlbmFudAAGdGVuYW50ACQ5ZjI1MjU1MS0zMDNmLTMzMDMtOWY5MC01ZmUyNjViOTcyNzS-71TeFdrerQ"
      + management_zone = "test-mz-2"
        name            = "alert-profile-2"
        # (1 unchanged attribute hidden)
    }

Plan: 0 to add, 2 to change, 0 to destroy.

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

dynatrace_alerting.alert-profile["profile2"]: Modifying... [id=vu9U3hXa3q0AAAABABhidWlsdGluOmFsZXJ0aW5nLnByb2ZpbGUABnRlbmFudAAGdGVuYW50ACQ5ZjI1MjU1MS0zMDNmLTMzMDMtOWY5MC01ZmUyNjViOTcyNzS-71TeFdrerQ]
dynatrace_alerting.alert-profile["profile1"]: Modifying... [id=vu9U3hXa3q0AAAABABhidWlsdGluOmFsZXJ0aW5nLnByb2ZpbGUABnRlbmFudAAGdGVuYW50ACQ1NDI0YjU4ZS0xNWViLTMyMTQtYTUxZi03ZTA5MTRlYWYyYWW-71TeFdrerQ]
dynatrace_alerting.alert-profile["profile2"]: Modifications complete after 0s [id=vu9U3hXa3q0AAAABABhidWlsdGluOmFsZXJ0aW5nLnByb2ZpbGUABnRlbmFudAAGdGVuYW50ACQ5ZjI1MjU1MS0zMDNmLTMzMDMtOWY5MC01ZmUyNjViOTcyNzS-71TeFdrerQ]
dynatrace_alerting.alert-profile["profile1"]: Modifications complete after 0s [id=vu9U3hXa3q0AAAABABhidWlsdGluOmFsZXJ0aW5nLnByb2ZpbGUABnRlbmFudAAGdGVuYW50ACQ1NDI0YjU4ZS0xNWViLTMyMTQtYTUxZi03ZTA5MTRlYWYyYWW-71TeFdrerQ]

Apply complete! Resources: 0 added, 2 changed, 0 destroyed.
>
